don't blindly overwrite $LD_LIBRARY_PATH (which breaks ld.gold linker because it requires a sufficiently recent libstdc++.so)
author: Kenneth Hoste (HPC-UGent)
+++ ncbi-vdb-2.10.7/build/ld.linux.exe.sh	2020-06-08 16:42:51.922658824 +0200
@@ -91,8 +91,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS
--- ncbi-vdb-2.10.7/build/ld.linux.dlib.sh.orig	2020-05-22 16:58:49.000000000 +0200
+++ ncbi-vdb-2.10.7/build/ld.linux.dlib.sh	2020-06-08 16:43:13.663013183 +0200
@@ -83,8 +83,7 @@
     done
 
     # update LD_LIBRARY_PATH
-    unset LD_LIBRARY_PATH
-    export LD_LIBRARY_PATH="$LDIRS:$XDIRS"
+    export LD_LIBRARY_PATH="$LDIRS:$XDIRS:$LD_LIBRARY_PATH"
 
     # tack on libraries, finding as we go
     for LIB in $LIBS
